pkgs/development/tools/poetry2nix/poetry2nix/overrides/default.nix

```diff
@@ -214,7 +214,7 @@ lib.composeManyExtensions [
             nativeBuildInputs = with pkgs;
               (old.nativeBuildInputs or [ ])
                 ++ lib.optionals (lib.versionAtLeast old.version "4")
-                (with pkgs.rustPlatform; [ rust.rustc rust.cargo cargoSetupHook self.setuptools-rust ]);
+                [ pkgs.rustPlatform.cargoSetupHook self.setuptools-rust ];
```

Comment:
This file has a bit strange workflow and it needs to be upstreamed, too. 

https://github.com/nix-community/poetry2nix

Reply:
I take it that I won't touch it for now because I do not know much about it.